UPGRADE-6.6.md

```diff
@@ -1,3 +1,51 @@
+# 6.6.10.8
+With this change, the minimal search term length is now loaded from the config table instead of being retrieved from the `.env` file.
```

Comment:
here the headline is missing. it is from `changelog/_unreleased/2025-07-07-use-minimal-search-term-length-in-config-tables.md`

Reply:
done